.github/workflows/CICD.yml

```diff
@@ -997,6 +997,123 @@ jobs:
         name: toybox-result.json
         path: ${{ steps.vars.outputs.TEST_SUMMARY_FILE }}
 
+  coverage:
+    name: Code Coverage
+    runs-on: ${{ matrix.job.os }}
+    timeout-minutes: 90
+    env:
+      SCCACHE_GHA_ENABLED: "true"
+      RUSTC_WRAPPER: "sccache"
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - { os: ubuntu-latest  , features: unix, toolchain: nightly }
+          # FIXME: Re-enable macos code coverage
```

Comment:
Maybe explain what is missing

Reply:
The script is a bit "hardcoded" for linux for now, because of the default features, and for some reason it can not find "show-utils.sh" when running on macos.
I intend to make it work for macos as well, but in another PR, since this is already a big patch.